### ISSUE TITLE
"Locating Screen Memory" example needs clarification

### ISSUE THREAD

**Issue by binarycrusader** [MEGA65/mega65-user-guide#622]
The "Locating Screen Memory" in "VIC-IV VIDEO MEMORY" in the user's guide seems to need a small correction to the example.

However, given the exposition above it, I'm uncertain if I'm just misreading the example:

> The highest nibble of the address is stored in the lower nibble of $D063, which will always be zero for addresses up to 005.FFFF.

https://github.com/MEGA65/mega65-user-guide/blob/d1ae32d741d7e133c4919d620ed27977ee8836aa/memory.tex#L833

> ; For completeness , clear bits 0-3 of SCRNPTRMB. The
> ; MEGA65 does not have VIC-accessible memory beyond
> ; 005 .FFFF, so this is not necessary if the previous
> ; value pointed to a valid address.
> LDA $D063
> **AND #$F0 ; clear bits 0 -3**
> STA $D063

**Comment by lgblgblgb** [comment]
I _think_ it means that MEGA65 can access more video RAM but on the current mega65-core models, there isn't more memory implemented (though VIC-IV is ready to access more in the future maybe - without any register layout/etc change).

**Comment by binarycrusader** [comment]
Yes, I assumed it was an attempt to future-proof, but I think the text might be clearer if it also incporated the code example's advice. Thanks so much!
